zk/src/org/zkoss/zk/ui/impl/DesktopImpl.java

```diff
@@ -167,7 +170,19 @@ public class DesktopImpl implements Desktop, DesktopCtrl, java.io.Serializable {
 	//don't create it dynamically because PageImp._ip bind it at constructor
 	private transient Execution _exec;
 	/** A list of ScheduleInfo; must be thread safe */
```

Comment:
remove this comment which is not true now

Reply:
_schedInfos is a thread-safe cache as well.